sdk/iot/hub/src/az_iot_hub_client_twin.c

```diff
@@ -12,15 +12,17 @@
 #include <_az_cfg.h>
 
 static const uint8_t az_iot_hub_client_twin_hashtag = '#';
-static const az_span az_iot_hub_client_twin_request_id_suffix = AZ_SPAN_LITERAL_FROM_STR("?$rid=");
-
-static const az_span az_iot_hub_twin_response_sub_topic
-    = AZ_SPAN_LITERAL_FROM_STR("$iothub/twin/res/");
-static const az_span az_iot_hub_twin_get_pub_topic = AZ_SPAN_LITERAL_FROM_STR("$iothub/twin/GET/");
+static const uint8_t az_iot_hub_client_twin_question = '?';
+static const uint8_t az_iot_hub_client_twin_equals = '=';
+static const az_span az_iot_hub_client_request_id_span = AZ_SPAN_LITERAL_FROM_STR("$rid");
+static const az_span az_iot_hub_twin_topic_prefix = AZ_SPAN_LITERAL_FROM_STR("$iothub/twin/");
+static const az_span az_iot_hub_twin_response_sub_topic = AZ_SPAN_LITERAL_FROM_STR("res/");
+static const az_span az_iot_hub_twin_get_pub_topic = AZ_SPAN_LITERAL_FROM_STR("GET/");
+static const az_span az_iot_hub_twin_version_prop = AZ_SPAN_LITERAL_FROM_STR("$version");
 static const az_span az_iot_hub_twin_patch_pub_topic
-    = AZ_SPAN_LITERAL_FROM_STR("$iothub/twin/PATCH/properties/reported/");
+    = AZ_SPAN_LITERAL_FROM_STR("PATCH/properties/reported/");
 static const az_span az_iot_hub_twin_patch_sub_topic
-    = AZ_SPAN_LITERAL_FROM_STR("$iothub/twin/PATCH/properties/desired/");
+    = AZ_SPAN_LITERAL_FROM_STR("PATCH/properties/desired/");
 
 AZ_NODISCARD az_result az_iot_hub_client_twin_response_subscribe_topic_filter_get(
```

Comment:
According to the last email from @CIPop summarizing the C99 meeting discussion, we need to change the functions to be something like: az_iot_hub_client_twin_response_get_subscribe_topic, or similar (get is not at the end). Is he doing that on a separate PR?

Reply:
I think those should be changed as we edit the files. AKA I can do that here

Reply:
Scratch that I will do it for the parse here and the other names in another PR. Actually I can do that with the `char*` updates.